central/policy/service/customresource/custom_resource.go

```diff
@@ -72,7 +74,9 @@ func toDNSSubdomainName(name string) string {
 
 	// If the result is empty or invalid after processing, return a default value
 	if len(name) == 0 {
-		name = "rhacs.default-policy-name"
+		// Get a random alphanumeric string from the first segment of a uuid
+		randAlphaNumerics := strings.Split(uuid.NewV4().String(), "-")
+		name = fmt.Sprintf("rhacs.%s", randAlphaNumerics[0])
```

Comment:
Can we use a dot instead of a dash here to be consistent with the non-default value?